js/scripts.js

```diff
@@ -0,0 +1,24 @@
+﻿function checkOnclickEvent(param) {
+  console.log(param);
+}
+
+var inputElem = document.getElementById('js-inputValue'),
+    imageElem = document.getElementById('js-image');
+    add = document.getElementById("addElem");
+    list = document.getElementById("list");
+
+    imageElem.addEventListener('click', function(e) {
+  checkOnclickEvent('image was clicked');
+});
+
+inputElem.addEventListener('keypress', function(e) {
```

Comment:
better use `keyup` event, `keypress` is still working even if you press and hold your button